src/main.rs

```diff
@@ -524,19 +523,13 @@ impl Fastmod {
     }
 
     fn print_diff<'a>(&mut self, diffs: &[DiffResult<&'a str>]) {
```

Comment:
DiffResult comes from the `diff` crate. Does it really make sense for us to be using both `diff` and `similar` for diffing? Perhaps cleaning this up further is out of scope for this PR, but it seems like we should at least open an issue to get back to having only one diffing crate?